msgcntr/messageforums-app/src/java/org/sakaiproject/tool/messageforums/ui/DiscussionForumBean.java

```diff
@@ -38,6 +38,9 @@
 import org.sakaiproject.api.app.messageforums.ui.DiscussionForumManager;
 import org.sakaiproject.api.app.messageforums.ui.UIPermissionsManager;
 
+import org.sakaiproject.time.cover.TimeService;
```

Comment:
Could you avoid the use of cover service here? Maybe loading through the ComponentManager service.